WellLog.Lib/Helpers/ByteHelpers.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace WellLog.Lib.Helpers
+{
+    public static class ByteHelpers
+    {
+        public static bool GetBit(this byte b, byte mask)
```

Comment:
nit: rename to GetBitFromMask

Reply:
GetMaskedBit would be more appropriate

Reply:
@TomMarks, the bit itself is not masked.  the bit mask is used to reveal the value of a bit within a byte.

@RyanDawkins, perhaps GetBitUsingMask would be the description for which you are looking?

Reply:
Yeah I think that function name describes the behavior.

Reply:
I looked at this one first, because why not start with the fifth change?  I put some simpler implementations below.